### PR TITLE
Support writing points via UDP

### PR DESCRIPTION
**Pull request by errm**
I added support to use the UDP interface...

### CODE REVIEW

**Comment by DavidHuie** [comment]
:+1:

**Comment by cimm** [comment]
Was wondering if the gem supported UDP as that's a must have for us, thanks! @toddboom the last release is around 8 months old, is there someone maintaining this gem?

**Comment by toddboom** [comment]
@cimm I'm going to play ruby gem cleanup today - I'll try to get a final release for v0.8 out ASAP and then move towards cleaning up the API for v0.9.

**Comment by cimm** [comment]
@toddboom Thank you for cleaning up and merging the pull requests, much appreciated.

@errm I pulled master to play with the UDP support today and feel it's an immature pull requests. Didn't realize that earlier, sorry. Here are some remarks:

The UDP client initializer signature is different from the HTTP client one. It's way more limited and the port is required (it's optional in the HTTP client). You can't specify a database as in the HTTP client. This feels like a quick hack. Shouldn't we decide to load or the UDP or the HTTP client instead of abusing the HTTP one to load the UDP client? https://github.com/influxdb/influxdb-ruby/blob/425cbb9556554ac050734c7025e3e5477313aed2/lib/influxdb/client.rb#L70

A generic `rescue`  is probably a bad idea, it will eat everything that might go wrong without the user noticing. What's its purpose? I feel it's better to let the calling method handle the error and decide what to do with it? https://github.com/influxdb/influxdb-ruby/blob/4345a5d3cc0d158064ccb45f36500fad217b85de/lib/influxdb/udp_client.rb#L13

The `influxdb.write_point(name,data)` call alternately returns an integer and nil. But none of the values ever reach the server.

TL;DR: This pull request is a good start to support UDP but is not production ready and should, IMHO, not be on master.

**Comment by mohitnegi** [comment]
@toddboom, included influxdb in my Gemfile. the gem does not contain UDP support, when i look into the branch, the master has the latest udp_client.rb file, but in  the tag 0.1.8, it's missing.
UDP needed urgently
